Modules/_ctypes/callproc.c

```diff
@@ -2015,6 +2066,7 @@ PyMethodDef _ctypes_module_methods[] = {
      "dlopen(name, flag={RTLD_GLOBAL|RTLD_LOCAL}) open a shared library"},
     {"dlclose", py_dl_close, METH_VARARGS, "dlclose a library"},
     {"dlsym", py_dl_sym, METH_VARARGS, "find symbol in shared library"},
+    {"get_interp", get_interp, METH_NOARGS},
```

Comment:
Documentation is missing for this new API.

Reply:
Also, if you introduce a new API, please use Argument Clinic.

Reply:
Is there some way to add it as an internal function? So we don't expose it as a public API

Reply:
Well, you could prefix it with an underscore.